src/clients/aws_client_factory.py

```diff
@@ -100,17 +101,26 @@ def get_ec2_client(self, account: Account, role: Optional[str] = None) -> AwsEC2
     def get_route53_boto_client(self, account: Account, role: str) -> BaseClient:
         return self._get_client("route53", account, role)
 
-    def get_route53_client(self, account: Account, role: Optional[str] = None) -> AwsRoute53Client:
-        return AwsRoute53Client(self.get_route53_boto_client(account, role or self._config.route53_role()))
+    def get_route53_client(self, account: Account) -> AwsRoute53Client:
+        return AwsRoute53Client(
+            boto_route53=self.get_hosted_zones_client(account),
+            iam=self.get_iam_client(account),
+            logs=self.get_logs_client(account, region="us-east-1"),
```

Comment:
Is it always this `us-east-1` region for route53? Could we maybe make this into a constant so it describes exactly why it is hard coded?

Reply:
yes, it's always us-east-1 region for route53, and that is why it makes more sense to hardcode it..